presto-main/src/main/java/com/facebook/presto/operator/unnest/UnnestOperator.java

```diff
@@ -52,7 +60,14 @@
         private final boolean withOrdinality;
         private boolean closed;
 
-        public UnnestOperatorFactory(int operatorId, PlanNodeId planNodeId, List<Integer> replicateChannels, List<Type> replicateTypes, List<Integer> unnestChannels, List<Type> unnestTypes, boolean withOrdinality)
+        public UnnestOperatorFactory(
```

Comment:
nit: would you move this refactoring into a separate commit?

Reply:
Sure. Moved this refactor and method reordering into a separate commit "Refactor UnnestOperator"